services/user-management/src/main/java/com/foodie/user/controllers/PermissionController.java

```diff
@@ -30,7 +31,8 @@ public PermissionController(PermissionService permissionService) {
     @PostMapping(produces = "application/json", consumes = "application/json")
     public ResponseEntity<FoodieBaseResponse> createPermission(@Valid @RequestBody Permission permission) {
         if (permission == null) {
```

Comment:
This check is probably redundant, the request will be if valid if body is not set.

Reply:
A null object will pass through which is why I introduced the check

Reply:
If you have required fields inside `Permission` it will ensure that null gets returned as `BadRequest`